### DISCUSSION TITLE
Change in-game pp counter to use the latest pp calculation algorithm

### DISCUSSION THREAD

**Discussion by ikkue** [ppy/osu#21101] [Ideas] [answered]
The title says it all already. I imported a few scores set in osu!(stable) into osu!(lazer) and watched all the way through the replays, and the pp counter both in-game and at the results screen does not match the actual pp achieved shown on the web.

**Comment by peppy** [comment]
lazer uses the latest development version. if anything it is newer than your expectation.

**Reply by smoogipoo** [reply]
While this is true, there haven't yet been any changes since the last deploy.

**Comment by smoogipoo** [comment, marked as answer]
Please provide replays/scores.

**Reply by ikkue** [reply]
I remember there being more when going through the top 20 most pp plays (which is almost all set by mrekk unsurprisingly), but this is the one that I remembered clearly.

**Map:** [THE ORAL CIGARETTES - ReI [Reflection] (Nevo)](https://osu.ppy.sh/beatmapsets/1465638#osu/3010178)
**Score:** https://osu.ppy.sh/scores/osu/4073299632 by mrekk
| web | lazer (2022.1101.0)|
|:---:|-------|
| ![image](https://user-images.githubusercontent.com/11213135/199929992-52040301-7be9-4641-8eb8-0a70f5bb7f1e.png) | ![osu_2022-11-04_15-34-00](https://user-images.githubusercontent.com/11213135/199930105-b915743f-8b32-4e4b-9c4d-5e2205a9b4da.png) |

Upon closer inspection, not only do the pp not match, but the 300s, 100s, and accuracy.

**Reply by integerrr** [reply]
Some replays are known to be inconsistent and this is tracked elsewhere e.g. #11311

**Reply by ikkue** [reply]
> Some replays are known to be inconsistent and this is tracked elsewhere e.g. #11311

Ah, I didn't see that! Thank you.
